staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -82,8 +83,10 @@ type Store struct {
 	// curl GET /apis/group/version/namespaces/my-ns/myresource
 	NewListFunc func() runtime.Object
 
-	// QualifiedResource is the pluralized name of the resource.
-	QualifiedResource schema.GroupResource
+	// DefaultQualifiedResource is the pluralized name of the resource.
```

Comment:
DefaultQualifiedResource is the pluralized name of the resource.
This field is used if there is no request info present in the context.
See qualifiedResourceFromContext for details.